tests/test_resource_tracker.py

```diff
@@ -157,6 +172,48 @@ def create_resource(rtype):
                                                 errno.ENOENT))
         assert re.search(expected, err) is not None
 
+    def test_resource_tracker_refcounting(self):
+        cmd = '''if 1:
+        import os
+        import tempfile
+        import time
+        from loky.backend import resource_tracker
+
+        resource_tracker.VERBOSE=True
+
+        tmpfile = tempfile.NamedTemporaryFile(delete=False)
+        filename = tmpfile.name
+        tmpfile.close()
+        assert os.path.exists(filename)
+
+        from loky.backend.resource_tracker import _resource_tracker
+        _resource_tracker.register(filename, "file")
+        _resource_tracker.register(filename, "file")
+
+        # forget all information about the resource, but do not try to remove
+        # it
+        _resource_tracker.unregister(filename, "file")
+        time.sleep(1)
+        assert os.path.exists(filename)
+
+        _resource_tracker.register(filename, "file")
+        _resource_tracker.register(filename, "file")
+        _resource_tracker.maybe_unlink(filename, "file")
+        time.sleep(1)
+        assert os.path.exists(filename)
+
+        _resource_tracker.maybe_unlink(filename, "file")
+        for _ in range(100):
+            if not os.path.exists(filename):
+                break
+            time.sleep(.1)
+        else:
+            raise AssertionError("%s was not unlinked in time"  % filename)
+        '''
+        p = subprocess.Popen([sys.executable, '-E', '-c', cmd])
+        p.wait()
+        assert p.returncode == 0
```

Comment:
`subprocess.check_call([sys.executable, '-E', '-c', cmd])` should do the same in one line if I am not mistaken.